.github/workflows/deploy.yml

```diff
@@ -0,0 +1,111 @@
+# Expected secrets
+# GOOGLE_PLAY_CLOUD_PROJECT - Google Cloud project associated with Google Play
+# GOOGLE_PLAY_SERVICE_ACCOUNT - Email address of service account
+# GOOGLE_PLAY_WORKLOAD_IDENTITY_PROVIDER - Workload identity provider to generate temporary service account key
+# SIGNING_KEYSTORE_BASE_64 - The signing key for the app
+# SIGNING_KEYSTORE_PASSWORD - The password for SIGNING_KEYSTORE_BASE_64
+# SIGNING_KEY_ALIAS - The key alias inside SIGNING_KEYSTORE_BASE_64
+# SIGNING_KEY_ALIAS_PASSWORD - The password for the key alias
+
+name: Deploy
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
(Reminder to myself that we need to configure branch protection rules prior to merging this!)

Reply:
Specifically, we need at least one review approval from someone other than the PR author, otherwise a dev could steal the secrets by approving their own malicious PR.

Reply:
I set up the branch protection rule